tests/adapters/test_record_adapter.py

```diff
@@ -155,3 +155,42 @@ def test_filters_or_records_validation(mock_record):
     with pytest.raises(ValueError):
         validate_filters_or_records([mock_record, ('Number', 'equals', 1)])
 
+
+def test_bulk_modify(mock_swimlane, mock_app, mock_record):
```

Comment:
Short doc string with what you're targeting in the test beyond just which function you're dealing with.